flexget/api/app.py

```diff
@@ -24,6 +29,18 @@
 
 logger = logger.bind(name='api')
 
+if TYPE_CHECKING:
+    from typing import TypedDict
```

Comment:
wont this crash before py3.8?

Reply:
That's why this and all other usages of TypedDict are in `if TYPE_CHECKING` blocks.
These parts will only be considered by mypy and never run by the python interpreter, therefore not crashing.

Reply:
Ah cool, didn't know that.